From eda36feb2b7eef8347f579c3c8590a8f09017cf0 Mon Sep 17 00:00:00 2001
From: Mark Janes <mark.a.janes@intel.com>
Date: Fri, 5 Apr 2019 11:39:18 -0700
Subject: [PATCH] intel/tools: Remove redundant definitions of INTEL_DEBUG

INTEL_DEBUG is declared extern and defined in gen_debug.c

Reviewed-by: Kenneth Graunke <kenneth@whitecape.org>
---
 src/intel/common/gen_disasm.c | 2 --
 src/intel/tools/i965_disasm.c | 2 --
 2 files changed, 4 deletions(-)

diff --git a/src/intel/common/gen_disasm.c b/src/intel/common/gen_disasm.c
index 4f835c19883..656613cd931 100644
--- a/src/intel/common/gen_disasm.c
+++ b/src/intel/common/gen_disasm.c
@@ -28,8 +28,6 @@
 
 #include "gen_disasm.h"
 
-uint64_t INTEL_DEBUG;
-
 struct gen_disasm {
     struct gen_device_info devinfo;
 };
diff --git a/src/intel/tools/i965_disasm.c b/src/intel/tools/i965_disasm.c
index 79434e6462a..5fe4afa35ff 100644
--- a/src/intel/tools/i965_disasm.c
+++ b/src/intel/tools/i965_disasm.c
@@ -29,8 +29,6 @@
 #include "compiler/brw_eu.h"
 #include "dev/gen_device_info.h"
 
-uint64_t INTEL_DEBUG;
-
 /* Return size of file in bytes pointed by fp */
 static size_t
 i965_disasm_get_file_size(FILE *fp)
